assets/content/static/footer.html

```diff
@@ -6,44 +6,36 @@ <h4 class="mb-0 font-weight-light">Get in touch with using any of these platform
         </div>
         <div class="col-lg-6 text-lg-center btn-wrapper">
             <a target="_blank" href="https://twitter.com/goasksef"
-                class="btn btn-neutral btn-icon-only btn-twitter btn-round btn-lg mt-2" data-toggle="tooltip"
-                data-original-title="Follow us">
+                class="btn btn-neutral btn-icon-only btn-twitter btn-round btn-lg mt-2" data-toggle="tooltip" title="Follow us">
```

Comment:
How about "Follow us on Twitter" ? or just "Twitter"

Reply:
Yes I thought that @jayasanka-sack but I put the one in the early code. Then shall I put as Twitter?

Reply:
> How about "Follow us on Twitter" ? or just "Twitter"

Follow us on Twitter is too long right? @jayasanka-sack

Reply:
I think it is ok with the Follow us on Twitter also @jayasanka-sack

Reply:
let's go with the name. ex: "Twitter", "Facebook", etc.

Reply:
Sure @jayasanka-sack I'll update and let you know

Reply:
Thanks @YoshithaRathnayake ! ❤️